content/sensu-go/5.21/reference/assets.md

```diff
@@ -93,25 +93,62 @@ Follow the steps outlined in [Contributing Assets for Existing Ruby Sensu Plugin
 For further examples of Sensu users who have added the ability to use a community plugin as an asset, see [this Discourse post](https://discourse.sensu.io/t/how-to-use-the-sensu-plugins-kubernetes-plugin/1286).
 {{% /notice %}}
 
-### Asset path
+### Default cache directory
+
+system  | sensu-backend                         | sensu-agent
+--------|---------------------------------------|-------------
+Linux   | `/var/cache/sensu/sensu-backend`      | `/var/cache/sensu/sensu-agent`
+Windows | N/A                                   | `C:\ProgramData\sensu\cache\sensu-agent`
+
+If the requested asset is not in the local cache, it is downloaded from the asset URL.
+The Sensu backend does not currently provide any storage for assets.
+Sensu expects assets to be retrieved over HTTP or HTTPS.
+
+### Example asset structure
+
+{{< code shell >}}
+sensu-example-handler_1.0.0_linux_amd64
+├── CHANGELOG.md
+├── LICENSE
+├── README.md
+└── bin
+  └── my-check.sh
+└── lib
+└── include
+{{< /code >}}
+
+## Asset path
 
 When you download and install an asset, the asset's local path on disk is exposed to asset consumers via environment variables.
-This allows you to retrieve the asset's path from a check, hook, handler, or mutator either as an environment variable or from a check or hook using a custom function for token substitution, `assetPath`.
+This allows you to retrieve the asset's path as an [environment variable][14] from a check, handler, hook, or mutator or using a [custom function for token substitution][17], `assetPath`, from a check or hook.
 
-For example, if you included a configuration file in the `include` directory of the asset [`sensu-plugins-windows`][4], you can reference the asset from your check in either of these ways:
+### Environment variable for asset path
 
-- `$SENSU_PLUGINS_WINDOWS/include/config.yaml`
-- `${{assetPath "sensu-plugins-windows"}}/include/config.yaml`
+If you included a configuration file in the `include` directory of the asset [`sensu-plugins-windows`][4], you can use the asset path environment variable to reference the asset from your check, handler, hook, or mutator:
```

Comment:
```suggestion
For each runtime asset, a corresponding environment variable will be available in the `command` context. The environment variable name is generated by capitalizing the asset name, replacing any special characters with underscores, and appending the `_PATH` suffix. The value of the variable will be the path on disk where the asset build has been unpacked.
```

I think the example usecase "If you included a configuration file in the `include` directory of the asset..." belongs in a different section where we describe "why you might want this".